social-distancing/libs/detectors/edgetpu/MobileNetSSD.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+import time
+from threading import Thread
+from queue import Queue
+import numpy as np
+import wget 
+
+from tflite_runtime.interpreter import load_delegate
+from tflite_runtime.interpreter import Interpreter
+
+class Detector():
```

Comment:
Can you please document this class? It looks very similar to Neuralet source code.